src/components/image/index.tsx

```
@@ -187,8 +192,8 @@ class Image extends PureComponent<Props, State> {
   };
 
   renderSvg = () => {
-    const {source, ...others} = this.props;
-    return <SvgImage data={source} {...others}/>;
+    const {source, recorderTag, ...others} = this.props;
```

Are we not setting the default to 'mask' here?